cfgov/jinja2/v1/browse-basic/index.html

```diff
@@ -14,7 +15,12 @@
 
 {% block content_main %}
     {% for block in page.header -%}
-        {% if loop.index == 1 %}
+        {% if 'featured_content' in block.block_type %}
```

Comment:
Sorry, seeing now this is the main content. How will we guarantee the title and intro come before the FCM?

Also, leave a not for BEWDs to add the flush sides modifier like they've done elsewhere.